CompanyEmployees.Infrastructure.Persistence/Configurations/AttendanceConfiguration.cs

```diff
@@ -0,0 +1,28 @@
+using CompanyEmployees.Core.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace CompanyEmployees.Infrastructure.Persistence.Configurations;
+
+public class AttendanceConfiguration : IEntityTypeConfiguration<Attendance>
+{
+    public void Configure(EntityTypeBuilder<Attendance> builder)
+    {
+        builder.HasKey(a => a.Id);
+
+        builder.HasOne(a => a.Employee)
```

Comment:
To align with application conventions, you could add an `inverse navigation property` in the `Employee` entity and use `.WithMany(e => e.Attendances)` instead of leaving it blank. This improves relationship clarity and navigability.

**Suggested change**
```suggestion
builder.HasOne(a => a.Employee)
    .WithMany(e => e.Attendances)
    .HasForeignKey(a => a.EmployeeId)
    .OnDelete(DeleteBehavior.Cascade);
```